recipes/_master_slurm_config.rb

```diff
@@ -48,6 +48,15 @@
   mode '0755'
 end
 
+cookbook_file '/etc/systemd/system/slurmctld.service' do
+  source 'slurmctld.service'
+  owner 'root'
+  group 'root'
+  mode '0755'
+  action :create
+  only_if { ::File.exist?('/bin/systemctl') }
```

Comment:
CentOS6 and alinux didn't adopt systemd. Is this fix only for ubuntu1604 and centos7?

Reply:
Why not use if node['init_package'] = systemd